system/lib/libc/musl/arch/emscripten/bits/alltypes.h

```diff
@@ -90,7 +90,13 @@ typedef long suseconds_t;
 
 
 #if defined(__NEED_pthread_attr_t) && !defined(__DEFINED_pthread_attr_t)
+#ifdef __EMSCRIPTEN__
```

Comment:
The file is `emscripten/bits/alltypes.h`, so already Emscripten-specific, isn't this `#ifdef` redundant?